app/services/phone_verification.rb

```diff
@@ -18,58 +19,42 @@ def initialize(phone:, code:, locale: nil)
     @locale = locale
   end
 
-  # rubocop:disable Style/GuardClause
   def send_sms
-    unless start_request.success?
-      raise VerifyError.new(
-        code: error_code,
-        message: error_message,
-        status: start_request.response_code,
-        response: start_request.response_body
-      )
-    end
+    tries ||= 2
```

Comment:
Shouldn't this be 'tries = 2'?

Reply:
Nope. It would never get to zero in that case and would keep retrying infinitely.

Reply:
Ah retry